workbench-core/accounts/src/services/projectService.ts

```diff
@@ -56,20 +62,143 @@ export default class ProjectService {
     }
   }
 
+  // TODO--delete after dynamic Authz
+  private _mockGetUserGroups(): string[] {
+    return [];
+  }
+
   /**
    * List projects
    *
+   * @param request - the optional fields to list projects
+   * @param user - authenticated user listing the projects
    * @returns Project entries in DDB
    */
-  public async listProjects(): Promise<{ data: Project[] }> {
-    const queryParams = {
+  // TODO--add filter support
+  public async listProjects(
+    user: AuthenticatedUser,
+    request?: ListProjectsRequest
+  ): Promise<{ data: Project[]; paginationToken: string | undefined }> {
+    const pageSize = request ? request.pageSize : undefined;
+    let paginationToken = request ? request.paginationToken : undefined;
+    // Get user groups--TODO implement after dynamic AuthZ
+    // const userGroupsForCurrentUser: string[] = await this._dynamicAuthorizationService.getUserGroups(user.id);
+    const userGroupsForCurrentUser: string[] = this._mockGetUserGroups(); // mock so the tests work
+
+    // If no group membership, return
+    if (_.isEmpty(userGroupsForCurrentUser)) {
+      return { data: [], paginationToken: undefined };
+    }
+
+    // If IT Admin, return all with pagination
+    if (userGroupsForCurrentUser.length === 1) {
+      if (userGroupsForCurrentUser[0] === 'ITAdmin') {
```

Comment:
Suggest extracting ITAdmin as const/enum

Reply:
This is a good idea but until we know what Dynamic AuthZ provides us, I am going to keep this value as a string. I will make a note to revisit during integration with Dynamic AuthZ. That package may provide an enum of special type.

Reply:
I'd still suggest extracting enum and replace it if authz has something similar